code/game/objects/effects/decals/cleanable/humans.dm

```diff
@@ -217,16 +197,27 @@
 				GLOB.bloody_footprints_cache["exited-[blood_state]-[Ddir]"] = bloodstep_overlay = image(icon, "[blood_state]2", dir = Ddir)
 			add_overlay(bloodstep_overlay)
 
-	alpha = BLOODY_FOOTPRINT_BASE_ALPHA+bloodiness
+	alpha = min(BLOODY_FOOTPRINT_BASE_ALPHA + (255 - BLOODY_FOOTPRINT_BASE_ALPHA) * bloodiness / (BLOOD_ITEM_MAX / 2), 255)
 
 
 /obj/effect/decal/cleanable/blood/footprints/examine(mob/user)
 	. = ..()
-	if(shoe_types.len)
-		. += "You recognise the footprints as belonging to:\n"
-		for(var/shoe in shoe_types)
-			var/obj/item/clothing/shoes/S = shoe
-			. += "[icon2html(initial(S.icon), user)] Some <B>[initial(S.name)]</B>.\n"
+	if((shoe_types.len + species_types.len) > 0)
+		. += "You recognise the footprints as belonging to:"
+		for(var/sole in shoe_types)
+			var/obj/item/clothing/item = sole
+			var/article = initial(item.gender) == PLURAL ? "Some" : "A"
```

Comment:
Try using "a(n)"

Reply:
I already tried this and it doesn't work, since `item` here is only a typepath and not an instanciated object.
```
. += "[icon2html(initial(item.icon), user, initial(item.icon_state))] [article] <B>[initial(item.name)]</B>."
. += "[icon2html(initial(item.icon), user, initial(item.icon_state))] \an <B>[item]</B>."
. += "[icon2html(initial(item.icon), user, initial(item.icon_state))] \an <B>[initial(item.name)]</B>."
. += "[icon2html(initial(item.icon), user, initial(item.icon_state))] \an [item]."
. += "[icon2html(initial(item.icon), user, initial(item.icon_state))] \an [initial(item.name)]."
```
This code results in:
![image](https://user-images.githubusercontent.com/2332094/91077824-04253a00-e642-11ea-9fef-38668e0c3c19.png)